ee/models/event_definition.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from posthog.models.event_definition import EventDefinition
+
+
+class EnterpriseEventDefinition(EventDefinition):
```

Comment:
Is this the best way? Should we do an explicit `OneToOne` relationship instead? That way we make sure we're not duplicating objects.

Reply:
I think tim also had that concern but it seems like it is using a one to one field? https://docs.djangoproject.com/en/3.2/topics/db/models/#multi-table-inheritance

Reply:
Perfect! Sounds good then